### PR TITLE
EPAS 17 UTL_FILE Updates

### PR DESCRIPTION
**Pull request by gvasquezvargas**
## What Changed?

https://enterprisedb.atlassian.net/browse/DB-2614 
First pass at implementing changes to UTL_FILE for EPAS 17.

### CODE REVIEW

File: product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_bip_guide/03_built-in_packages/20_utl_file.mdx
Comment (gvasquezvargas) [inline product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_bip_guide/03_built-in_packages/20_utl_file.mdx:1]: @Nishant-at-EDB, could you please review the modifications to this file as per https://enterprisedb.atlassian.net/browse/DB-2614 ?

File: product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_bip_guide/03_built-in_packages/20_utl_file.mdx [hunk 1]
```diff
@@ -416,7 +434,19 @@ The `FOPEN` function opens a file for I/O.
...
 
- Maximum size of a line in characters. In read mode, an exception is thrown if you try to read a line exceeding `max_linesize`. In write and append modes, an exception is thrown if you try to write a line exceeding `max_linesize`. The end-of-line characters aren't included in determining if the maximum line size is exceeded. This behavior isn't compatible with Oracle databases. Oracle counts the end-of-line characters.
+ Maximum size of a line in characters. The default is 1024 characters.
+ In read mode, an exception is thrown if you try to read a line exceeding `max_linesize`. 
+ In write and append modes, an exception is thrown if you try to write a line exceeding `max_linesize`. 
+ The end-of-line characters are included when determining if the maximum line size is exceeded.
+ This procedure is compatible with Oracle.
+
+ Supported modes are: 
+    - `a` &mdash; Append to file.
```
Comment (nidhibhammar) [inline product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_bip_guide/03_built-in_packages/20_utl_file.mdx:444]:
```suggestion
    - `a` &mdash; append to file.
```

Comment (nidhibhammar) [reply product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_bip_guide/03_built-in_packages/20_utl_file.mdx:444]: same suggestions for below lines

File: product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_bip_guide/03_built-in_packages/20_utl_file.mdx [hunk 2]
```diff
@@ -449,7 +480,19 @@ FOPEN_NCHAR(location VARCHAR2, filename VARCHAR2, open_mode VARCHAR2, max_linesi
...
```
Comment (nidhibhammar) [inline product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_bip_guide/03_built-in_packages/20_utl_file.mdx:490]: same comments as above

**Comment by nidhibhammar** [comment]
@Nishant-at-EDB, Please review these doc updates and let us know if any changes are needed.

**Review by dwicinas — COMMENTED** [review]
The changes I reviewed look good but a more technical review is required.

**Pull request merged**